playground/src/components/custom-views/demo-custom-view.jsx

```diff
@@ -0,0 +1,87 @@
+import { useEffect, useState } from 'react';
+import {
+  CustomViewShell,
+  useMcQuery,
+} from '@commercetools-frontend/application-shell';
+import { useApplicationContext } from '@commercetools-frontend/application-shell-connectors';
+import { GRAPHQL_TARGETS } from '@commercetools-frontend/constants';
+import Constraints from '@commercetools-uikit/constraints';
+import DataTable from '@commercetools-uikit/data-table';
+import LoadingSpinner from '@commercetools-uikit/loading-spinner';
+import Spacings from '@commercetools-uikit/spacings';
+import Text from '@commercetools-uikit/text';
+import { CUSTOM_VIEW_ID } from './custom-panel-demo';
+import FetchChannelsQuery from './fetch-channels.ctp.graphql';
+
+const channelsColumns = [
+  {
+    key: 'key',
+    label: 'Key',
+  },
+  {
+    key: 'name',
+    label: 'Name',
+  },
+];
+
+function ChannelsCustomView() {
+  const { projectName, locale } = useApplicationContext((context) => ({
+    projectKey: context.project.key,
+    projectName: context.project.name,
+    locale: context.user.locale,
+  }));
+
+  const { data, error, loading } = useMcQuery(FetchChannelsQuery, {
```

Comment:
I don't understand why but this query is failing with this error:
```json
{
  "errors": [
    {
      "message": "Cannot query field \"channels\" on type \"Query\".",
      "extensions": {
        "code": "GRAPHQL_VALIDATION_FAILED"
      }
    }
  ]
}
```

I doubled checked the headers we're sending and they are the same as the ones sent from the Merchant Center channels view.

Reply:
This issue has been fixed here: dbdbf6d6a3a3a00353cfaf5b338ad2950a8029f4

Thanks @emmenko for helping in finding the root cause 🙇